pkg/scaffold/api.go

```diff
@@ -174,19 +175,20 @@ func (api *API) scaffoldV2() error {
 			return err
 		}
 
-		if !api.resourceExists() {
-			api.project.Resources = append(api.project.Resources,
-				input.Resource{Group: r.Group, Version: r.Version, Kind: r.Kind})
+		// Only save the resource in the config file if it didn't exist
+		if !api.config.HasResource(api.Resource) {
+			// Add the new resource
+			api.config.Resources = append(api.config.Resources,
+				config.Resource{Group: r.Group, Version: r.Version, Kind: r.Kind})
 
-			// If the --force was used to re-crete a resource that was created before then,
-			// the PROJECT file will not be updated.
-			if err := saveProjectFile("PROJECT", api.project); err != nil {
```

Comment:
@Adirio see that the previous code does not allow we save it in any place in any part of the code.

Reply:
It does load it from a different place, so keeping generic load, save and check functions seems reasonable:

https://github.com/Adirio/kubebuilder/blob/6474bc287b76bc2d8e76e51016db0403828abd12/pkg/scaffold/scaffold.go#L135

Reply:
My point is: the current impl do not allow it ends up in situations that it was called in diff places and with diff paths which could cause issues such as the project has 2 projects files.

Reply:
You mean like storing where it was loaded from and saving in the same place?

Reply:
I mean that with the changes in this PR we can config.Save(string) in any part of the code with any path informed so it can bring to the project awkward situations and issues which with the current implementations are not possible.

Reply:
Will make a `Save` method, I like the idea.